sdk/java-sdk/src/main/scala/kalix/javasdk/impl/ProxyInfoHolder.scala

```diff
@@ -58,15 +57,10 @@ class ProxyInfoHolder(system: ExtendedActorSystem) extends Extension {
 
     this._proxyHostname = Some(chosenProxyName)
 
-    // if portOverride is filled, we choose it.
-    // Otherwise we use the announced one.
-    // Note: with old proxy versions `proxyInfo.proxyPort` will default to 0
-    this._chosenPort = _portOverride.getOrElse(proxyInfo.proxyPort)
-
     this._identificationInfo = proxyInfo.identificationInfo
 
     log.debug("Proxy hostname: [{}]", chosenProxyName)
-    log.debug("Proxy port to: [{}]", _chosenPort)
+    log.debug("Proxy port to: [{}]", proxyInfo)
```

Comment:
When running tests, this log will print the announced proxy port, even if later it can be overwritten.